README.md

```diff
@@ -33,6 +33,24 @@ In this example, we use WIMSE with HTTP Message Signatures to explore the use ca
                              └──────────────────┘     
 ``` 
 
+A sequence diagram of the steps is provided:
+
+```mermaid
+sequenceDiagram
+    participant ID as Identity Server (minispire)
```

Comment:
nit: I wonder if adding a line break (`<br>`) before "(minispire)" would work and render nicer.